episodes/01-basics.md

```diff
@@ -95,33 +143,71 @@ the same files concurrently.
 
 :::::::::::::::::::::::::
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
+## Version control and R files
+
+For code-like files like `.R` and `.Rmd` files, we can not use Google docs. The software and strategy to track changes in a project depends on the **file type**.
+
+- Google Docs’ version history tool is a Version control software optimized for single [non-plain text files](https://github.com/epiverse-trace/git-rstudio-basics/discussions/11) like Word/Doc files that works online.
+
+- `Git` is the Version control software optimized for [plain text files](https://en.wikipedia.org/wiki/Plain_text) that works offline. (Read: “What Not to Put Under Version Control” at [G. Wilson et al. 2017](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005510#sec014))
+
+Plain text files can be text, code, and data. Example for each of these are Markdown files (`.md`), R files (`.R`), and `.csv` or `.tsv` files, respectively.
+
+![Examples of non-plain and plain text files.](fig/plain-text-files.png)
+
+::::::::::::::::: callout
+
+We can use `Git` to track changes of these plain text files. However, for [plain text data files](https://www.r4epi.com/importing-plain-text-files.html) (like `.csv` and `.tsv`) we prefer different version control systems.
+
+:::::::::::::::::::::::::
+
+Plain text files like Markdown files (`.md`) and R files (`.R`) are integrated in Rmarkdown files (`.Rmd`) to generate manuscripts, websites, and R packages. These three products are outputs of **Open Science** projects, that leads to *Reproducible research* and *Sustainable software*.
+
+
+![We can increase the reproducibility of our Open science projects with version control sytems like `Git`. Text and final results can be connected and executable by Data and code. From: "Ciencia reproducible: qué, por qué, cómo" <https://www.revistaecosistemas.net/index.php/ecosistemas/article/view/1178>](fig/gradient-reproducibility.png)
+
+::::::::::::::::::: instructor
+
+### Concept map
+
+![Concept map for automatic version control.](fig/git-concept-03.drawio.png)
+
+::::::::::::::::::::::::::::::
+
+::::::::::::::::: discussion
+
+### Exercise!
+
+Tell us about your Open Science project and its file types!
+
+- Briefly share about one Open Science project in which they are involved or would like to start soon (e.g. thesis, current project, or work);
+- Identify the most relevant file types (`.pdf`, `.jpge`, `.csv`, `.xlsx`, `.R`, `.docx`, `.Rmd`) involved in it and classify them as non-plain or plain text files;
+- Discuss which ones can use a Version control software like Git? 
+
+::::::::::::::::::::::::::::
+
+::::::::::::::::::: instructor
+
+This exercise can be solved in the shared document of the training.
+
+::::::::::::::::::::::::::::::
+
 :::::::::::::::::::::::::::::::::::::::: keypoints
 
 - Version control is like an unlimited 'undo'.
```

Comment:
Might be better to change "unlimited" to "step-by-step"

Reply:
```suggestion
- Version control record changes you make "step-by-step".
```

Reply:
Agree, I wrote the action too in the sentence. The undo feature is covered explicitly in episode 05